backend/infrahub/core/attribute.py

```diff
@@ -599,6 +600,28 @@ class Boolean(BaseAttribute):
     type = bool
 
 
+# TODO: Add properties for different datetime format outputs to mimic IP property types for GraphQL
+class DateTime(BaseAttribute):
+    type = str
+
+    @classmethod
+    def validate_format(cls, value: Any, name: str, schema: AttributeSchema) -> None:
+        super().validate_format(value=value, name=name, schema=schema)
+
+        # If DateTime attribute is an empty string and is not required do not complete any further validation of format.
+        # This was added due to infrahub.core.integrity.object_conflict.conflict_recorder.py initialize_validator sets completed_at to a blank string
```

Comment:
I think you can remove this comment
the logic it describes seems straightforward and reasonable to me